apps/server/db/moduls/coupons.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require("mongoose");
+const couponsScheme = new mongoose.Schema({
```

Comment:
model schema start with capital char because it inherit class 

```suggestion
const Coupon = new mongoose.Schema({
```